src/scrapers/mizrahi.js

```diff
@@ -0,0 +1,119 @@
+import moment from 'moment';
+import {
+  SHEKEL_CURRENCY,
+  NORMAL_TXN_TYPE,
+  TRANSACTION_STATUS,
+  ISO_DATE_FORMAT,
+} from '../constants';
+import { BaseScraperWithBrowser, LOGIN_RESULT } from './base-scraper-with-browser';
+import { fetchPostWithinPage } from '../helpers/fetch';
+
+const BASE_URL = 'https://www.mizrahi-tefahot.co.il';
+const LOGIN_URL = `${BASE_URL}/he/bank/Pages/Default.aspx`;
+const AFTER_LOGIN_BASE_URL = 'https://mto.mizrahi-tefahot.co.il/ngOnline/index.html#/main/uis/osh/p428/';
```

Comment:
If I remember correctly, Mizrahi allow users to customize the first page post login. which means that the url will end differently and `#/main/uis/osh/p428/` is not guaranteed. 

can you please check the settings page and confirm if this is possible, if so you can always use a regexp and rely only on the start of the link before the hash

Reply:
My solution no longer looks very beautiful.
I need to navigate to the "Osh" (עו"ש) url to sniff the transactions request.
But if I don't `waitForNavigation`, the login not complete.

Maybe should I do this in `fetchData`?

Reply:
> @baruchiro why aren't we using a regular string here? would also be great to reuse `https://mto.mizrahi-tefahot.co.il` as something like `BASE_APP_URL`.

@eshaham Did you notice that?

Reply:
@baruchiro sababa, but
1. you can also create regex strings with regular strings. I usually prefer to either do that, or if I feel this format is necessary, I only use it inside the `match` (or `replace`) functions. I dislike using a regex string in this format out of the context of its usage
2. you can still create the `BASE_APP_URL` constant, right?